jenkins_jobs/modules/hipchat_notif.py

```diff
@@ -82,6 +82,8 @@
 import jenkins_jobs.errors
 import jenkins_jobs.modules.base
 
+import pkg_resources
```

Comment:
Put this in the same place as in upstream, ie. after `import logging`. It'll save some troubles once we decide to merge new JJB version from upstream to our fork.